[2021-03-04 21:44:35,580] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T21:44:31.403251+00:00 [queued]>
[2021-03-04 21:44:35,605] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T21:44:31.403251+00:00 [queued]>
[2021-03-04 21:44:35,606] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 21:44:35,606] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-03-04 21:44:35,606] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 21:44:35,644] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): load_data> on 2021-03-04T21:44:31.403251+00:00
[2021-03-04 21:44:35,647] {standard_task_runner.py:54} INFO - Started process 7172 to run task
[2021-03-04 21:44:35,682] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'clean_data', 'load_data', '2021-03-04T21:44:31.403251+00:00', '--job_id', '95', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_clean.py', '--cfg_path', '/tmp/tmpr_a5uw_h']
[2021-03-04 21:44:35,685] {standard_task_runner.py:78} INFO - Job 95: Subtask load_data
[2021-03-04 21:44:35,748] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: clean_data.load_data 2021-03-04T21:44:31.403251+00:00 [running]> 6623e80cd770
[2021-03-04 21:44:35,787] {data_clean.py:65} INFO - ****** NOAH starting load data
[2021-03-04 21:44:35,857] {logging_mixin.py:112} WARNING - /home/airflow/.local/lib/python3.6/site-packages/pandas/core/generic.py:2612: UserWarning: The spaces in these column names will not be changed. In pandas versions < 0.14, spaces were converted to underscores.
  method=method,
[2021-03-04 21:44:36,210] {data_clean.py:77} INFO - ****** NOAH 2 ending load data
[2021-03-04 21:44:36,213] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-03-04 21:44:36,224] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=clean_data, task_id=load_data, execution_date=20210304T214431, start_date=20210304T214435, end_date=20210304T214436
[2021-03-04 21:44:40,561] {local_task_job.py:102} INFO - Task exited with return code 0
